static/css/base.css

```diff
@@ -339,4 +339,4 @@ thead tr {
   -moz-border-radius: var(--border-radius);
   -webkit-border-radius: var(--border-radius);
```

Comment:
These prefixed vendor properties are unnecessary, as `border-radius` has been a standard CSS property in all major browsers for a long time.